docs/ca-design.rst

```diff
@@ -253,9 +253,9 @@ defined in use cases layer.
     Class that is checking if the package is newer or not than the package currently available
     in Fedora using mdapi system. Inherits from `validator.py`.
 
-  * **pdc.py**
+  * **retired.py**
```

Comment:
There should be one validator per external system (it makes it easier to replace them when needed). In this case it would be better to update the Pagure validator and add a new key `retired` to output dictionary.

Reply:
Added `retired` as a new key in the output dictionary under the `pagure` validator. 
But I'm not too happy about having two request calls in one function. It complicates the testing and does not feel right. I'll try to figure out how to make it better.